platform/common/driver/vault.go

```diff
@@ -12,19 +12,26 @@ import (
 	"github.com/hyperledger-labs/fabric-smart-client/platform/common/utils/collections"
 )
 
+type (
+	PKey     = string
```

Comment:
Why is it called `PKey`? What is the `P` standing for?

Reply:
I guess property, but I chose this name because it was the most common name for this type of variable. for the metadata keys i used `MKey`

Reply:
I see. I find it a bit misleading when looking at the code what uses now `driver.PKey` as it is not really clear what this type is meant for. Is it something vault specific or is is a Public Key? :)

Reply:
if rwset is a map, pkey is the key and rawvalue is the value. we could also name it `RWSKey` and `RWSValue`. I would do it in another separate pr to be honest once we agree on something